core/Cargo.toml

```diff
@@ -50,3 +50,7 @@ serde = ["multihash/serde-codec", "dep:serde", "libp2p-identity/serde"]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 rustc-args = ["--cfg", "docsrs"]
+
+# lint
+[lints]
+workspace = true
```

Comment:
Can you ensure there are newlines at the end of files? See the red marker here in the diff. You can probably configure your editor to automatically add them.

Reply:
Sure, Just now made a commit, Please review. Sorry for the delay.